js/src/components/google-ads-account-card/google-ads-account-card.js

```diff
@@ -10,22 +16,37 @@ import AuthorizeAds from './authorize-ads';
 import { GOOGLE_ADS_ACCOUNT_STATUS } from '.~/constants';
 
 export default function GoogleAdsAccountCard() {
-	const { google, scope } = useGoogleAccount();
-	const { googleAdsAccount } = useGoogleAdsAccount();
+	const {
+		google,
+		scope,
+		isResolving: isResolvingGoogleAccount,
+	} = useGoogleAccount();
+	const { googleAdsAccount, isResolving: isResolvingGoogleAdsAccount } =
+		useGoogleAdsAccount();
 
-	if ( ! google || ! googleAdsAccount ) {
+	if ( isResolvingGoogleAccount || isResolvingGoogleAdsAccount ) {
```

Comment:
@eason9487 I tried on the R1.3 PR and I could not reproduce. Can you kindly check on your end please?
https://github.com/woocommerce/google-listings-and-ads/pull/2270

Thanks!

Reply:
Commenting on the issue in this PR is because it happens with this PR, but it doesn't mean the issue also happens on the subsequent PR.

Since the merge base of this PR is `develop`, even though #2270 will not have the same issue, it doesn't mean that this PR with that issue is suitable to be merged directly to `develop` without fixing it.